tokio/tests/rt_panic.rs

```diff
@@ -0,0 +1,105 @@
+#![warn(rust_2018_idioms)]
+#![cfg(feature = "full")]
+
+use lazy_static::lazy_static;
+use std::error::Error;
+use std::panic;
+use std::sync::{Arc, Mutex, Once};
+use tokio::runtime::{Builder, Handle, Runtime};
+
+fn test_panic<Func: FnOnce() + panic::UnwindSafe>(func: Func) -> Option<String> {
+    lazy_static! {
+        static ref SET_UP_PANIC_HOOK: Once = Once::new();
+        static ref PANIC_MUTEX: Arc<Mutex<()>> = Arc::new(Mutex::new(()));
+        static ref PANIC_FILE: Mutex<String> = Mutex::new(String::new());
+    }
```

Comment:
We already have the dependencies to do this. There's no reason to add lazy_static.
```suggestion
use parking_lot::{const_mutex, Mutex};

static SET_UP_PANIC_HOOK: Once = Once::new();
static PANIC_MUTEX: Mutex<()> = const_mutex(());
static PANIC_FILE: Mutex<String> = const_mutex(String::new());
```

Reply:
Thank you! I was hoping to get a comment like this.